setup/dusty.rb

```diff
@@ -1,113 +1,38 @@
-    sha256 "0e2b62b497bd5f0afebc002eda4d90df9d209c30ef257e8673c90a6b5c119d62"
-  end
-
-  resource "PrettyTable" do
-    url "https://pypi.python.org/packages/source/P/PrettyTable/prettytable-0.7.2.tar.bz2"
-    sha256 "853c116513625c738dc3ce1aee148b5b5757a86727e67eff6502c7ca59d43c36"
-  end
-
-  resource "docopt" do
-    url "https://pypi.python.org/packages/source/d/docopt/docopt-0.6.2.tar.gz"
-    sha256 "49b3a825280bd66b3aa83585ef59c4a8c82f2c8a522dbe754a8bc8d08c85c491"
-  end
-
-  resource "pathtools" do
-    url "https://pypi.python.org/packages/source/p/pathtools/pathtools-0.1.2.tar.gz"
-    sha256 "7c35c5421a39bb82e58018febd90e3b6e5db34c5443aaaf742b3f33d4655f1c0"
-  end
-
-  resource "argh" do
-    url "https://pypi.python.org/packages/source/a/argh/argh-0.26.1.tar.gz"
-    sha256 "06a7442cb9130fb8806fe336000fcf20edf1f2f8ad205e7b62cec118505510db"
-  end
-
-  resource "watchdog" do
-    url "https://pypi.python.org/packages/source/w/watchdog/watchdog-0.8.3.tar.gz"
-    sha256 "7e65882adb7746039b6f3876ee174952f8eaaa34491ba34333ddf1fe35de4162"
-  end
-
-  resource "python-dateutil" do
-    url "https://pypi.python.org/packages/source/p/python-dateutil/python-dateutil-2.4.2.tar.gz"
-    sha256 "3e95445c1db500a344079a47b171c45ef18f57d188dffdb0e4165c71bea8eb3d"
-  end
-
-  resource "pync" do
-    url "https://pypi.python.org/packages/source/p/pync/pync-1.6.1.tar.gz"
-    sha256 "85737aab9fc69cf59dc9fe831adbe94ac224944c05e297c98de3c2413f253530"
-  end
-
-  resource "Schemer" do
-    url "https://pypi.python.org/packages/source/S/Schemer/Schemer-0.2.7.tar.gz"
-    sha256 "94bc1f69da4abf1b8f5e64cfe58c82406d3c4f7963a7cad7cd42aac109638415"
-  end
-
-  resource "gitdb" do
-    url "https://pypi.python.org/packages/source/g/gitdb/gitdb-0.6.4.tar.gz"
-    sha256 "a3ebbc27be035a2e874ed904df516e35f4a29a778a764385de09de9e0f139658"
-  end
-
-  resource "GitPython" do
-    url "https://pypi.python.org/packages/source/G/GitPython/GitPython-1.0.1.tar.gz"
-    sha256 "9c88c17bbcae2a445ff64024ef13526224f70e35e38c33416be5ceb56ca7f760"
-  end
-
-  def install
-    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python2.7/site-packages"
-    %w[backports.ssl_match_hostname six websocket-client requests docker-py PyYAML smmap PrettyTable docopt pathtools argh watchdog python-dateutil pync Schemer gitdb GitPython].each do |r|
-      resource(r).stage do
-        system "python", *Language::Python.setup_install_args(libexec/"vendor")
-      end
-    end
-
-    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
-    system "python", *Language::Python.setup_install_args(libexec)
-
-    bin.install Dir[libexec/"bin/*"]
-    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
-  end
-
-  test do
-    Language::Python.each_python(build) do |python, _version|
-      system python, "-c", "import dusty"
-    end
-  end
+cask 'dusty' do
+  version '0.6.5'
```

Comment:
If we need to update this each release, can you update the release checklist in our docs?